amida-tech/indaba-client
# Feature/INBA-497 All Users Delete Checks

#### What's this PR do?
Display a prompt before deleting users from the All Users page.

#### Related JIRA tickets:
[INBA-497](https://jira.amida-tech.com/browse/INBA-497)

#### How should this be manually tested?
1. Create users in three data states:
   1. Has no tasks assigned in any project
   1. Has tasks assigned but has not answered questions in any survey for them
   1. Has answered survey questions for an assigned task
1. Delete the users from the All Users page and check that a prompt appears with [correct language](https://paper.dropbox.com/doc/Indaba-Conditional-Limits-ublG6ZqtvItRL1LL924Zd#:uid=093953831492899346329275&h2=Conditional-Delete)
   - NOTE expect deletes to fail if users are in groups ([INBA-440](https://jira.amida-tech.com/browse/INBA-440))

#### Any background context you want to provide?
This branch sits on #248, review that PR first

#### Screenshots (if appropriate):
